phi/tools/slack.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
```

Comment:
wrap in try/except Import error. see: https://github.com/phidatahq/phidata/blob/main/phi/tools/duckdb.py#L6-L9